runtime-light/stdlib/rpc/rpc-buffer.h

```diff
@@ -43,7 +43,7 @@ class RpcBuffer : private vk::not_copyable {
   }
 
   void reset(size_t pos) noexcept {
-    php_assert(pos >= 0 && pos <= size());
```

Comment:
Why do you think it should be changed this way?

Reply:
As I remember clang-format gives an error that size_t can't be less than 0